python/lsst/sims/utils/healpyUtils.py

```diff
@@ -172,3 +172,55 @@ def healbin(ra, dec, values, nside=128, reduceFunc=np.mean, dtype=float):
     """
     return _healbin(np.radians(ra), np.radians(dec), values, nside=nside,
                     reduceFunc=reduceFunc, dtype=dtype)
+
+
+def moc2array(data, uniq, nside=128, reduceFunc=np.sum, density=True, fillVal=0.):
+    """Convert a Multi-Order Coverage Map to a single nside HEALPix array. Useful
+    for converting maps output by LIGO alerts. Expect that future versions of
```

Comment:
It's not clear to me what this is returning.  Is it returning a list of healpix id's corresponding to the area specified by the MOC array, or it is returning a numpy array of some value whose density over healpixels is being manipulated?